src/pages/index.js

```diff
@@ -257,361 +291,348 @@ const Page = () => {
-              <IndexCardLink
-                to="/biganimal/latest/using_cluster/06_demonstration_oracle_compatibility/"
-                className="nested-link"
-              >
-                Oracle SQL Compatibility
-                <span className="new-thing" title="Interactive Demo">
-                  Demo
-                </span>
-              </IndexCardLink>
-            </IndexCard>
-
-            <IndexCard iconName={iconNames.KUBERNETES} headingText="Kubernetes">
-              <IndexCardLink to="/postgres_distributed_for_kubernetes/latest/">
-                EDB Postgres Distributed for Kubernetes
-              </IndexCardLink>
+              <BannerWideLink to="/jdbc_connector/latest">JDBC</BannerWideLink>
+              <BannerWideLink to="/net_connector/latest">.NET</BannerWideLink>
+              <BannerWideLink to="/ocl_connector/latest">OCL</BannerWideLink>
+              <BannerWideLink to="/odbc_connector/latest">ODBC</BannerWideLink>
 
-              <IndexCardLink to="/postgres_for_kubernetes/latest/">
-                EDB Postgres for Kubernetes
-              </IndexCardLink>
+              <BannerDivider headingText="Data Adapters" />
 
-              <IndexCardLink to="/supported-open-source/cloud_native_pg/">
-                CloudNativePG
-              </IndexCardLink>
-            </IndexCard>
+              <BannerWideLink to="/pgbouncer/latest">PgBouncer</BannerWideLink>
+              <BannerWideLink to="/pgpool/latest">pgPool-II</BannerWideLink>
 
-            <IndexCard
-              iconName={iconNames.DRIVES}
-              headingText="Backup & Recovery"
-            >
-              <IndexCardLink to="/supported-open-source/barman/">
-                Barman
-              </IndexCardLink>
+              <BannerDivider headingText="Data Wrappers" />
+              <BannerWideLink to="/hadoop_data_adapter/latest">
+                Hadoop
+              </BannerWideLink>
+              <BannerWideLink to="/mongo_data_adapter/latest">
+                Mongo
+              </BannerWideLink>
+              <BannerWideLink to="/mysql_data_adapter/latest">
+                MySQL
+              </BannerWideLink>
+            </BannerWideSubCard>
+          </BannerCard>
 
-              <IndexCardLink
-                to="/supported-open-source/barman/single-server-streaming/"
-                className="nested-link"
-              >
-                Single Server Streaming
-                <span
-                  className="new-thing"
-                  title="Walk through an interactive demo in Katacoda"
-                >
-                  Demo
-                </span>
-              </IndexCardLink>
-              <IndexCardLink to="/supported-open-source/pgbackrest/">
-                pgBackRest
-              </IndexCardLink>
-            </IndexCard>
+          <BannerWideSubCard
+            iconName={iconNames.HANDSHAKE}
+            headingText="Third Party Integrations"
```

Comment:
Same here. 
Other banner headings of the same level have a link and are clickable. This one is clickable but does not take you anywhere. Should we add a linkk for consistency? Or remove the clickability of the button to avoid confusion.